Clases.py

```diff
@@ -21,26 +21,93 @@ def ver_elementos(self):
 bolsita.ver_elementos()
 
 
-class Persona:
-    def __init__(self, nombre, apellido, edad):
+class Persona():
+    def __init__(self, nombre, apellido, dni, edad):
         self.nombre = nombre
         self.apellido = apellido
+        self.dni = dni
         self.edad = edad
-
-    def cumpleanios(self):
-        self.edad += 1
-
-    def comotellamas(self):
-        print("Me llamo ", self.nombre, " ", self.apellido)
-
-
-franco = Persona("Franco", "Leiva", 28)
-paula = Persona("Paula", "Montesino", 32)
-yesica = Persona("Yesica", "Suarez", 22)
-paola = Persona("Paola", "Tabare", 22)
-paola.cumpleanios()
-paola.comotellamas()
-
+        
+    def habla(self, decir):
+        print(decir)
+        
+fran = Persona("Franco", "Leiva", 33333333, 29)
+fran.habla("Tenemos que ir a comprar alimento para perros")
+    
+class Docente(Persona):
+    def __init__(self, nombre, apellido, dni, edad, nLegajo, cAlumnos, antiguedad):
+        Persona.__init__(self, nombre, apellido, dni, edad)
+        self.materia = ""
+        self.nLegajo = nLegajo
+        self.cAlumnos = cAlumnos
+        self.antiguedad = antiguedad
+        
+    def set_materia(self,materia):
+        if materia == "Geografía" or "Matemática" or "Programación":
```

Comment:
No sé si probaste este metodo con alguna materia no permitida, pero este condicional siempre da true porque no está haciendo lo que esperás (probalo en una terminal, fuera del if). Una forma correcta sería:
`if materia in ("Geografía", "Matemática", "Programación"):`